Gruntfile.js

```diff
@@ -292,10 +292,9 @@ module.exports = function(grunt) {
     },
 
     shell: {
-      'npm-install': {
-        command: 'node scripts/npm/check-node-modules.js'
+      'install-node-dependencies': {
+        command: 'yarn'
```

Comment:
It shouldn't be a problem. The following was run on the angular.js repo with npm & yarn cache cleared:
```
$ time yarn --ignore-scripts
yarn --ignore-scripts  28.22s user 21.07s system 125% cpu 39.178 total
$ time yarn --ignore-scripts
yarn --ignore-scripts  6.49s user 2.68s system 120% cpu 7.583 total
$ time yarn --ignore-scripts
yarn --ignore-scripts  0.87s user 0.07s system 70% cpu 1.321 total
$ time yarn --ignore-scripts
yarn --ignore-scripts  0.86s user 0.07s system 70% cpu 1.313 total
```